TrackletAlgorithm/VMRouterCM.h

```diff
@@ -280,9 +280,11 @@ void VMRouterCM(const BXType bx, BXType& bx_o,
 		AllStub<OutType> allstub = (disk2S) ? stubDisk2S.raw() : stub.raw();
 
 		// Write stub to all memory copies
-		for (int n = 0; n < nAllCopies; n++) {
+		if (nAllCopies > 0) {
```

Comment:
@aryd I don't understand why this `if` block is needed. Isn't the `for` loop skipped anyway if `nAllCopies` is nonpositive?

Reply:
I agree - I don't see why this should be needed - but if I leave it out I get:

ERROR: [SYNCHK 200-61] ../../../TrackletAlgorithm/AllStubMemory.h:86: unsupported memory access on variable 'memoriesAS.dataarray_.data_.V' which is (or contains) an array with unknown size at compile time.
INFO: [SYNCHK 200-10] 1 error(s), 1 warning(s).
ERROR: [HLS 200-70] Synthesizability check failed.

If you see a better fix let me know.